Makefile

```diff
@@ -1,4 +1,4 @@
-## Copyright 2016 Istio Authors
+## Copyright 2017 Istio Authors
```

Comment:
I can remove this. Do we have requirements on when/if this date should ever be changed?